content/Projects/Observability/thanos.md

```diff
@@ -1,3 +1,61 @@
 # Thanos
 
-> In progress.
+Thanos is a horizontally scalable, multi-tenant monitoring system in a form of distributed time series database that supports Prometheus data format.
```

Comment:
This doesn't seem like it should be part of this PR

Reply:
It is, just to show how to point to sub project proposal and show how project template could look on Thanos example.